### PR TITLE
add reference to logo and bg color

### PR DESCRIPTION
**Pull request by jmcampanini**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -321,6 +321,10 @@ It should be subscribed to the following events:
 * Status
 * Pull request review
 
+There is a [`logo.png`](https://github.com/palantir/policy-bot/blob/develop/logo.png)
+provided if you'd like to use it as the GitHub application logo. The background
+color is `#4d4d4d`
```
Comment (bluekeyes) [inline README.md]:
```suggestion
color is `#4d4d4d`.
```

**Review by bluekeyes — APPROVED** [review]

**Comment by jmcampanini** [comment]
lets see if we can get a status

**Comment by jmcampanini** [comment]
it works! and it disqualified @bluekeyes for co-authoring the suggestion. lol. gonna merge this in and them make it required :)

**Comment by jmcampanini** [comment]
nm. gonna release new version first so we get secure contexts first

**Pull request merged**